src/components/Article.jsx

```diff
@@ -28,6 +29,13 @@ export function Article({ setArticleNotFound }) {
       });
   }, []);
 
+  function incrementArticleVote(inc_votes) {
+    const payload = { inc_votes };
+    return apiClient
+      .patch(`articles/${article_id}`, payload)
+      .then((data) => console.log(data));
+  }
```

Comment:
You could extract this whole api call out into your client file and have all the axios requests in there. This helps particularly if a change is needed to all the endpoints. They are all in one place.